content/blog/2019/10/2019-10-04-vscode-caseStudy.adoc

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: "JCasC JSONSchema Case-Study Using VSCode"
+opengraph:
+  image: /images/logos/JCasC/JCasC.png
+tags:
+- community-bridge
+- JCasC
+- VSCode
+- community
+author: sladyn98
+---
+
+=== JSON Schema and VSCode
+
+==== Configuration-as-code plugin
+The Configuration as Code plugin has been designed as an opinionated way to configure Jenkins based on human-readable declarative configuration files. Writing such a file should be feasible without being a Jenkins expert, just translating into code a configuration process one is used to executing in the web UI.The plugin uses a schema to verify the files being applied to the Jenkins instance.
```

Comment:
One sentence per line, please. It is hard to suggest changes

Reply:
Done :+1: